src/osm_configurator/view/states/state_manager.py

```diff
@@ -473,7 +483,7 @@ def unlock_state(self):
 
     def freeze_state(self):
         """
-        This method freezes all frames that are currently active, and making then not interactable anymore.
+        This method freezes all frames that are currently active, and making then not intractable anymore.
```

Comment:
same thing, intractable is the wrong word!

Reply:
ok